From ff0b746187184bfe0ec083280362d69a6c2060ca Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 28 May 2019 15:00:32 +0300
Subject: [PATCH] [SW-4185] pll: rename HL_PLL_SKIP bypass value

Rename the HL_PLL_SKIP => HL_PLL_WA_INIT to better reflect the correct
usage scenario - which is to move the divisor output clock select to
reference clock before the PLL configuration (regardless the current
settings) and skip its move back to the configuration defined state
until the W/A execution time.

Change-Id: Iad84432c54167bd45ef418b4a48830d9f6b5b7d7
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index 7be7e70..81f2065 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -119,7 +119,7 @@
 			regmap_writel(HL_PLL_DIV_PLL, reg);
 			break;
 		case HL_PLL_REF_CLK:
-		case HL_PLL_SKIP:
+		case HL_PLL_WA_INIT:
 		default:
 			break;
 		}
diff --git a/common/pll.h b/common/pll.h
index f1662a4..6cb640e 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -37,7 +37,7 @@
 	HL_PLL_PLL_CLK,
 	HL_PLL_DIV_REF,
 	HL_PLL_DIV_PLL,
-	HL_PLL_SKIP,
+	HL_PLL_WA_INIT,
 };
 
 enum hl_pll_div_out {
